### DISCUSSION TITLE
Is there a widget like TabLayout

### DISCUSSION THREAD

**Discussion by AliIoT** [hecrj/iced#1255] [Q&A] [answered]
<img width="579" alt="image" src="https://user-images.githubusercontent.com/99813530/154851126-4a8fd659-d656-497c-8d68-84d460e12922.png">

**Comment by 90-008** [comment, marked as answer]
Perhaps try https://github.com/iced-rs/iced_aw#tabbar-and-tabs?

**Reply by goflutterjava** [reply]
thanks for your reply. I am looking for this widget too.
